src/main/java/picard/arrays/illumina/CompareGtcFiles.java

```diff
@@ -58,7 +58,7 @@
     // Ignored methods
     private static final List<String> IGNORED_METHODS = Arrays.asList(
         "getClass",
-        "getAutocallDate",
+        "getAutoCallDate",
```

Comment:
Blame the code reviewers for missing that one

Reply:
Or *should* have cut and pasted.